src/blog/2022/10/db-migraton-01.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Planned Database Migration
+subtitle: Moving to encrypted storage
+description: Details ppcoming maintence period
+date: 2022-10-13
+authors: ["ben-hardill"]
+---
+
+# Planned Database Migration
+
+As part of an on going security [review](https://flowforge.com/product/security/#data-at-rest) of the FlowForge Managed offering we discovered that the backend database was not using encrypted storage. In keeping with industry best practices we plan to migrate the database to a new instance using encrypted at rest storage.
+
+## Impact
+
+To do this cleanly we are going to need to schdule a short period of downtime for the core FlowForge application. During this downtime the following functions will not be available
+
+- Creating/deleting FlowForge Projects
+- Creating/deleting FlowForge Users
+- Deploying changes to flows running in Projects
+
+Existing FlowForge Projects will continue to run as normal during the downtime.
```

Comment:
From experience: We need to stress that self managed customers will not be impacted. 

I know it's a bit redundant, but we should mention it